build.gradle

```diff
@@ -161,6 +157,110 @@ project ('hdfs') {
     }
 }
 
+project('test:integration') {
```

Comment:
is it possible to define `integration` task in each binding gradle project instead of it being a separate project?